proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -342,10 +343,33 @@ message QueryConsumerValidatorsValidator {
   // The power of the validator used on the consumer chain
   int64 power = 3;
   // The rate to charge delegators on the consumer chain, as a fraction
-  string rate = 4 [
+  string consumer_commission_rate = 4 [
     (gogoproto.customtype) = "cosmossdk.io/math.LegacyDec",
     (gogoproto.nullable)   = false
     ];
+  // description defines the description terms for the validator
+  cosmos.staking.v1beta1.Description description = 5 [(gogoproto.nullable) = false];
+  // operator_address defines the address of the validator's operator
+  string operator_address = 6 [(cosmos_proto.scalar) = "cosmos.AddressString"];
+  // jailed defined whether the validator has been jailed from bonded status or not.
+  bool jailed = 7;
+  // status is the validator status (bonded/unbonding/unbonded).
+  cosmos.staking.v1beta1.BondStatus status = 8;
+  // The rate to charge delegators on the provider chain, as a fraction
+  string provider_commission_rate = 9 [
```

Comment:
nit: slightly nicer under `consumer_commission_rate` so both commission rates are next to each other.